recipes/octo-logger-cpp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conans import ConanFile, CMake
+from conan.tools.files import get
+
+
+class OctoLoggerCPPConan(ConanFile):
+    name = "octo-logger-cpp"
+    license = "MIT"
+    url = "https://github.com/ofiriluz/octo-logger-cpp"
+    homepage = "https://github.com/ofiriluz/octo-logger-cpp"
+    description = "Octo logger library"
+    topics = ("logging", "cpp")
+    generators = "cmake", "cmake_find_package_multi"
+    settings = "os", "compiler", "build_type", "arch"
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
+
+    def source(self):
+        get(self, **self.conan_data["sources"][str(self.version)], strip_root=True, destination=self._source_subfolder)
+
+    def requirements(self):
+        self.requires("catch2/3.1.0")
+        self.requires("fmt/9.0.0")
+        self.requires("trompeloeil/42")
```

Comment:
```suggestion
        self.requires("fmt/9.0.0")
```
no unit tests, please.

Reply:
fixed